packages/flab/stringify.js

```diff
@@ -0,0 +1,9 @@
+console.log(
+	'module.exports = '+
+	require("fs").readFileSync("/dev/stdin", "utf-8")
```

Comment:
Oh cool thanks @drewpc.

I'm tempted to use [`get-stdin`](https://www.npmjs.com/package/get-stdin), which is a bit more targeted to what this little one-liner needs to do.